### ISSUE TITLE
VRAM Usage

### ISSUE THREAD

**Issue by PixelovyLabyrintDev** [oobabooga/text-generation-webui#5338]
Hello, it might be a dumb question, but i dont think the models are using my gpu as they should, is it normal that they only use the VRAM and nothing else from the gpu? I thought 0% usage meant it didnt work but i found out its using all my VRAM and not the gpu itself. 
TL;DR: Is it normal that it only uses my VRAM and nothing more from my gpu?

**Comment by MaxLanc** [comment]
The default view for the gpu on task manager shows 3D, copy, encode and decode, and AI stuff makes use of the CUDA cores, and task manager idk how it separates/checks the different usage of the engines exactly but you need to change the views to CUDA and see it's actual use

![imagen](https://github.com/oobabooga/text-generation-webui/assets/95586248/69259125-a0ee-4856-8921-4a70c2d30e9a)
pic related

**Comment by MaxLanc** [comment]
And here in use, keep in mind that I'm using tensorcores as well and part of the model remains loaded on the CPU/RAM

![imagen](https://github.com/oobabooga/text-generation-webui/assets/95586248/b13ce0c0-2fe6-4896-8a66-aa92d38d2b75)

**Comment by sandyis** [comment]
@PixelovyLabyrintDev , Yes mostly , atleast on my system but only in ideal state VRAM is where GPU is loaded 
GPU will get used when you make actual queries 

It's not unusual for certain tasks to primarily use VRAM with low GPU utilization. While high VRAM usage indicates active memory access, low GPU utilization doesn't necessarily mean the GPU isn't working. 
VRAM vs. GPU:

VRAM: Dedicated memory on the GPU for storing textures, models, and other data needed for rendering. Think of it as the GPU's RAM.
GPU: The processing unit itself, responsible for complex calculations and rendering tasks.
Why Low GPU Utilization with High VRAM Usage?

Some tasks, like loading large datasets or textures, involve intensive memory access but minimal processing, leading to high VRAM usage but seemingly low GPU utilization.
The actual calculations might be happening efficiently within the GPU even if the overall utilization appears low.
